### ISSUE TITLE
question about provided models & results

### ISSUE THREAD

**Issue by zozobozo** [TRI-ML/packnet-sfm#12]
Hi,

Pretty cool work, thanks for sharing! 
I have some questions regarding the provided **KITTI** models (linked in the README file), and their performance:

1. Is the performance (RMSE etc.) of all the models reported w.r.t the improved-GT
(your [paper's](https://arxiv.org/pdf/1905.02693.pdf) table 3 refers to "Improved[42]")? 
2. is the last row (with "densified GT") means you're training also using the improved ground-truth from ref [42] to **train on**? (Generally - it would be nice if you couldr add a column saying which models have which supervision, just like in your table 3).
3. in your paper, the last row in table 3 is absolutely amazing: 3.293 [m] RMSE, in absolute scale, without any lidar ground-truth, only pose information. This makes the gap to supervised methods much smaller. Is this model one of the provided models here?

Thanks for the help!
Z.

**Comment by VitorGuizilini-TRI** [comment]
1) No, we report metrics with both GT depth maps. We chose to do so because most self-supervised methods use the sparse "original" GT, while supervised methods use the densified "improved" GT. Since we are comparing with supervised methods (even though we are self-supervised), we decided to include this evaluation as well.
2) Yes, that model is semi-supervised and it uses the densified "improved" GT at training time. 
3) Yes, that's "PackNet, Self-Supervised Scale-Aware, 384x1280, CS → K", but on the table it's evaluated with the sparse "original" GT, so the numbers are different. We will add numbers when evaluating with the densified "improved" GT as well, they should be close to what the paper reports.

I agree that this is confusing, and this confusion is very common in depth estimation papers. We will clarify these points in the README, thank you for that!

**Comment by zozobozo** [comment]
Ok, great, thanks for your quick answer! i indeed think that clarifying these points in the README would be helpful. I think your table 3 is doing a great job clarifying this long-standing confusion in depth-estimation papers.

I have one follow-up question - regarding the scale-aware training:
in the CVPR paper you have a velocity-loss, but i didn't find it in the repository. Is this training script + loss function missing? or are you training with loaded poses from kitti, calling [_get_pose](https://github.com/TRI-ML/packnet-sfm/blob/master/packnet_sfm/datasets/kitti_dataset.py#L366)?

Thanks again!
Z.

**Comment by VitorGuizilini-TRI** [comment]
Very good point, we still haven't added velocity supervision loss as an option at training time, will do it very soon!

**Comment by VitorGuizilini-TRI** [comment]
Just following up on that, we have added the velocity supervision loss to the repository.
